### ISSUE TITLE
toStream should push the initial value

### ISSUE THREAD

**Issue by NetanelBasal** [mobxjs/mobx-utils#82]
Hey,

I am starting to work on a library that connects mobx to Angular 2. I am using `fromStream` to make a bridge between Rx and mobx, but the problem is that `fromStream` does not push the initial value. 

What can we do about that? 

I can use the `startWith` operator, but it's not cool 🙈. 

```ts
this.counter = from(toStream(() => this._counterStore.counter)).startWith(this._counterStore.counter)
```

Thanks.

**Comment by NetanelBasal** [comment]
We can add the following code to support this:

```ts
export function toStream<T>(expression: () => T, fireImmediately = true): IObservableStream<T> {
    const computedValue = computed(expression);
    return {
        subscribe(observer: any): ISubscription {
            return {
                unsubscribe: computedValue.observe(
                    typeof observer === "function"
                        ? ( {newValue}: { newValue: T } ) => observer(newValue)
                        : ( {newValue}: { newValue: T } ) => observer.next(newValue)
                , fireImmediately)
            };
        },
        [observableSymbol()]: self
    };
}

```

**Comment by mweststrate** [comment]
@NetanelBasal agreed! interested in creating a PR including a unit test?

**Comment by mweststrate** [comment]
Released as 3.2.2 (sorry for the late follow up!)
